src/homework/day8/objectclasses/Bubble.java

```diff
@@ -26,24 +25,4 @@ public String getName() {
     public void setName(String name) {
         this.name = name;
     }
-
-    @Override
-    public String toString() {
-        return "Bubble{" +
-                "volume=" + volume +
-                ", name='" + name + '\'' +
-                '}';
-    }
-
-    @Override
-    public boolean equals(Object o) {
-        if (this == o) return true;
-        if (!(o instanceof Bubble bubble)) return false;
-        return getVolume() == bubble.getVolume() && getName().equals(bubble.getName());
-    }
-
-    @Override
-    public int hashCode() {
-        return Objects.hash(getVolume(), getName());
-    }
 }
```

Comment:
I don't understand why you delete toString | equals | hashCode, because it was condition of task "все поля private, конструктор с инициализацией всех полей, переписанные тустринг, хеш-код, иквалз, геттеров и сеттеров на каждое поле):
-- Bubble (поля int volume, String name)". 
By the way, it wasn't necessary to create additional class for 11 task, you can just did it in the main method in this class.